FrameworkInternals/deviceGenerators.py

```diff
@@ -39,9 +39,8 @@ def generateDeviceClass(context, *classList):
 	for aClass in classList:
 		generateOneDeviceClass(context, aClass)
 	
-def generateAllDevices():
+def generateAllDevices(context):

```

Comment:
Feel free to ignore this comment - it's highly subjective, probably a bit rant-y.

I've always found the variable name 'context' frustratingly opaque when trying to understand code (and, since I'm writing about annoying variable names, 'data' is another one). I just mean that on reading code, variables named context doesn't reveal much about what is contained within, or is what it's used for. I mean all code executes in some context (and deals with data for that matter). So the reader either has to search around up the call stack to see where context came from (or downwards, to see how/where it's consumed. Or both).

If you leave as is there will be no further noise from me on the subject. It's not a big deal.